src/Pages/kitchen_sink/kitchen_sink.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Navbar } from "../../components";
+
+const kitchen_sink = () => {
+  return (
+    <>
+      <Navbar />
+      <div className="flex justify-center items-center bg-gray-400 w-screen h-screen">
+        {/* logo */}
+        <p
+          className="
+            text-white
+            cursor-pointer
+            text-2xl
+            -tracking-wider 
+            leading-7 
+            font-noto 
+            font-bold
+            hover:text-red-700 
+            transition-colors
+            duration-200
+            delay-100
```

Comment:
Excellent use of tailwind classes, great to see such a broad use and you're using state based classes.

Just to highlight, if you're getting over comfortable with tailwind but wouldn't feel comfortable implementing directly in CSS, maybe write some custom CSS from time to time. You could use tailwind for things you absolutely know about and write CSS manually for things you're not sure about.

The point would be to ensure you're not losing learning & knowledge on the application and writing of CSS by having it extracted through tailwind. These things are made to help & support development but it's really healthy to ensure you're able to understand everything it's doing and why it's important.

Reply:
I've figured out how to declare my classes as a variable and use them so I'm not rewriting the classnames multiple times over. I figured out also how to only target certain children in my array to have different styling when rendered. Hope i'm making sense lol

`const linkClasses: string =
    "invisible md:visible text-white lg:text-green cursor-pointer text-xs leading-7 font-robo font-bold hover:text-red-700 transition-colors duration-200 delay-75 mr-5 lg:mr-10";
  const logoClasses: string =
    "text-white cursor-pointer text-2xl -tracking-wider leading-7 font-noto font-bold mr-10 lg:mr-2`

`<div>
        {/* pages links */}
        {Links.map((link, index) => (
          <NavLink
            key={index}
            to={link.to}
            className={({ isActive }) =>
              `${index === 0 ? logoClasses : linkClasses} ${
                isActive ? "text-red-700" : ""
              }`
            }
          >
            {link.label}
          </NavLink>
        ))}
      </div>`